jekyll/_cci2/server/installation/phase-3-execution-environments.adoc

```diff
@@ -0,0 +1,1217 @@
+----
+# With the base64-encoded additional config from above
+kubectl create secret generic nomad-autoscaler-secret \
+  --from-literal=secret.hcl=$ADDITIONAL_CONFIG
+----
+--
+
+
+[#iam-policy-for-nomad-autoscaler]
+===== IAM policy for Nomad Autoscaler
+
+[source, json]
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "autoscaling:CreateOrUpdateTags",
+                "autoscaling:UpdateAutoScalingGroup",
+                "autoscaling:TerminateInstanceInAutoScalingGroup"
+            ],
+            "Resource": "<<Your Autoscaling Group ARN>>"
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+                "autoscaling:DescribeScalingActivities",
+                "autoscaling:DescribeAutoScalingGroups"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+
+endif::env-gcp[]
+
+ifndef::env-aws[]
+
+[#gcp-service-account]
+==== GCP autoscaler service account
+Create a service account for Nomad Autoscaler. You may take **one** of the following approaches:
+
+[.tab.gcpautoscaler.CircleCI_creates_secret]
+--
+**Option 1:** CircleCI creates the secret.
+
+The CircleCI link:https://github.com/CircleCI-Public/server-terraform/tree/main/nomad-gcp[Nomad module] can create a service account and output a file with the JSON key. For this option, set the variable `nomad_auto_scaler = true`. You may reference the examples in the link for more details. The created service account key will be available in a file named `nomad-as-key.json`.
+--
+
+[.tab.gcpautoscaler.Use_Workload_Identity]
+--
+**Option 2:** Use Workload Identity.
+
+The CircleCI link:https://github.com/CircleCI-Public/server-terraform/tree/main/nomad-gcp[nomad module] can create a service account using link:/docs/server/installation/phase-1-prerequisites#enable-workload-identity-in-gke[Workload Identity] and out the email. Set the variables `nomad_auto_scaler = true` and `enable_workload_identity = true`.
+--
+
+[.tab.gcpautoscaler.You_create_secret]
+--
+**Option 3:** Create the Secret yourself.
+When creating the secret manually, an additional field is required - see the below example.
+
+[source, shell]
+----
+# Base64 encoded additional configuration field
+ADDITIONAL_CONFIG=dGFyZ2V0ICJnY2UtbWlnIiB7CiAgZHJpdmVyID0gImdjZS1taWciCiAgY29uZmlnID0gewogICAgY3JlZGVudGlhbHMgPSAiL2V0Yy9ub21hZC1hdXRvc2NhbGVyL2NyZWRzL2djcF9zYS5qc29uIgogIH0KfQo=
+kubectl create secret generic nomad-autoscaler-secret \
+  --from-literal=gcp_sa.json=<service-account> \
+  --from-literal=secret.hcl=$ADDITIONAL_CONFIG
+----
+
+When creating a Nomad GCP service account manually. The service account will need the role `compute.admin`. It will also need the role `iam.workloadIdentityUser` if using Workload Identity. This step is only required if you choose not to create the service account using Terraform.
```

Comment:
```suggestion
When creating a Nomad GCP service account manually, the service account will need the role `compute.admin`. It will also need the role `iam.workloadIdentityUser` if using Workload Identity. This step is only required if you choose not to create the service account using Terraform.
```